### PR TITLE
chore: edited license headers

### PR DESCRIPTION
**Pull request by eschrewe**
## Description
- changed license header copyright year, where necessary
- solves #528

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files
- [x] If helm chart has been changed, the chart version has been bumped to either next major, minor or patch level (compared to released chart).

### CODE REVIEW

**Comment by eschrewe** [comment]
Won't do chart version bumb here...

File: local/generate-keys.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 #
-# Copyright (c) 2022,2024 Volkswagen AG
-# Copyright (c) 2022,2024 Contributors to the Eclipse Foundation
+# Copyright (c) 2023 Volkswagen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (tom-rm-meyer-ISST) [inline local/generate-keys.sh:5]: Seems like a copy and paste issue :)
```suggestion
# Copyright (c) 2022 Volkswagen AG
# Copyright (c) 2022 Contributors to the Eclipse Foundation
```

File: local/init-wallets.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #
-# Copyright (c) 2022,2024 Volkswagen AG
-# Copyright (c) 2022,2024 Contributors to the Eclipse Foundation
+# Copyright (c) 2023 Volkswagen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (tom-rm-meyer-ISST) [inline local/init-wallets.sh:3]: Seems like a copy and paste issue :)
```suggestion
# Copyright (c) 2022 Volkswagen AG
# Copyright (c) 2022 Contributors to the Eclipse Foundation
```

File: local/vault/put-keys.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 #
-# Copyright (c) 2022,2024 Volkswagen AG
-# Copyright (c) 2022,2024 Contributors to the Eclipse Foundation
+# Copyright (c) 2023 Volkswagen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (tom-rm-meyer-ISST) [inline local/vault/put-keys.sh:5]: Seems like a copy and paste issue :)
```suggestion
# Copyright (c) 2022 Volkswagen AG
# Copyright (c) 2022 Contributors to the Eclipse Foundation
```

File: local/cleanup.sh [hunk 4]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 #
-# Copyright (c) 2022,2024 Volkswagen AG
-# Copyright (c) 2022,2024 Contributors to the Eclipse Foundation
+# Copyright (c) 2023 Volkswagen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```
Comment (tom-rm-meyer-ISST) [inline local/cleanup.sh:5]: Seems like a copy and paste issue :)
```suggestion
# Copyright (c) 2022 Volkswagen AG
# Copyright (c) 2022 Contributors to the Eclipse Foundation
```

**Review by tom-rm-meyer-ISST — CHANGES_REQUESTED** [review]
Thanks a lot! Found 4 places where 2023 has been placed instead of 2024.

Added suggestion so that you can also add them to a commit as a batch, via files tab :)

Please check :)

**Review by tom-rm-meyer-ISST — APPROVED** [review]
LGTM, thanks for raising!

**Pull request merged**